SCClassLibrary/Common/Audio/bela/BELAUGens.sc

```diff
@@ -68,10 +70,12 @@ DigitalIn : UGen {
  */
 DigitalOut : UGen {
     *ar { arg digitalPin = 0, output=0, writeMode=0, mul=1.0, add=0.0;
-        ^this.multiNew('audio', digitalPin, output, writeMode ).madd(mul,add)
+        this.multiNew('audio', digitalPin, output, writeMode ).madd(mul,add);
+		^0.0;
```

Comment:
something weird with indentation. You should use `tools/clang-format.py` to lint your files prior to commit

Reply:
Sorry, the problem is that clang-format doesn't format .sc files, and these files are all in spaces, while all .sc files in main SuperCollider are in tabs. I'll put those lines in spaces as well, then maybe we change all of them to tabs at once before merge with main sc?

Reply:
yeah we made a mess the moment we started editing these ... what I do in these cases is I try to keep the convention of the surrounding code. I thought the linter would take care of these, but clearly it doesn't, thanks for letting me know and fixing it.